build.gradle

```diff
@@ -1,3 +1,5 @@
+import java.util.regex.Pattern
```

Comment:
I prefer to not have imports in Gradle configs.
More over this is Groovy, so there is a way to deal with regular expressions without Java API: https://www.baeldung.com/groovy-pattern-matching